packages/fs-detectors/src/detect-builders.ts

```diff
@@ -339,7 +339,7 @@ export async function detectBuilders(
       warnings.push({
         code: 'conflicting_files',
         message:
-          'When using Next.js, it is recommended to place Node.js Serverless Functions inside of the `pages/api` (provided by Next.js) directory instead of `api` (provided by Vercel).',
+          'When using Next.js, it is recommended to place Node.js Functions inside of the `pages/api` (provided by Next.js) directory instead of `api` (provided by Vercel). Other (non-Node.js) Functions should still go in the `api` directory.',
```

Comment:
```suggestion
          'When using Next.js, it is recommended to place JavaScript Functions inside of the `pages/api` (provided by Next.js) directory instead of `api` (provided by Vercel). Other languages (such as Python) should still go in the `api` directory.',
```

Maybe we use JavaScript since both Node.js vs Edge Runtime should both be handled by Next.js

This message is starting to get really confusing now that we have `app` dir which can have Route Handlers at any path.

Reply:
```suggestion
          'When using Next.js, it is recommended to place JavaScript/TypeScript Functions inside of the `pages/api` (provided by Next.js) directory instead of `api` (provided by Vercel). Other languages (Python, Go, etc) should still go in the `api` directory.',
```

Makes sense to me! Should we go ahead and mention TypeScript here as well?